x-pack/plugins/enterprise_search/public/applications/analytics/components/analytics_collection_view/analytics_collection_view.tsx

```diff
@@ -34,9 +48,14 @@ export const collectionViewBreadcrumbs = [
 
 export const AnalyticsCollectionView: React.FC = () => {
   const { fetchAnalyticsCollection } = useActions(FetchAnalyticsCollectionLogic);
+  const { fetchAnalyticsCollectionDataViewId } = useActions(AnalyticsCollectionDataViewIdLogic);
   const { analyticsCollection, isLoading } = useValues(FetchAnalyticsCollectionLogic);
+  const { dataViewId } = useValues(AnalyticsCollectionDataViewIdLogic);
   const { id, section } = useParams<{ id: string; section: string }>();
   const { navigateToUrl } = useValues(KibanaLogic);
```

Comment:
Looking at kibanaLogic, looks like you have access to the `application` (ApplicationStart) here too. If you do, you wont need to use the `useKibana` hook to get the application.

Using the useValues(KibanaLogic) helps clean up the code (not needing to jest mock the public plugin)